README.md

```diff
@@ -21,7 +21,7 @@ To install the plugin manually using source code from this repository:
 
 ### Shortcodes
 
-1. Paywall: [lnd paywall amount=1 memo="product id 1"]Your content[/lnd]
+1. Chest: [lnd chest amount=1 memo="product id 1"]Your content[/lnd]
```

Comment:
Do you mind if we stick with 'paywall' ? I think its a more accurate description of the feature..

Reply:
I would like to implement the paywall as extra/additional shortcode (with filter). A paywall is a barrier for me which I have to overcome so that I can read the content behind the paywall. The treasure chest can appear several times on a page and behind it special contents/treasures can be hidden. For example a continuative video. Or links to higher resolution photographs.